### PR TITLE
Use /etc/passwd to resolve UIDs

### PR DESCRIPTION
**Pull request by mcpherrinm**

### CODE REVIEW

File: ownership.go [hunk 1]
```diff
@@ -50,40 +49,34 @@ func NewOwnership(username, groupname string) (Ownership, error) {
...
 	}
 
 	return gid, nil
 }
 
-func lookupGidInFile(groupname string, file *os.File) (uint32, error) {
+func lookupIDInFile(name, fileName string) (uint32, error) {
+	file, err := os.Open(fileName)
+	if err != nil {
+		return 0, fmt.Errorf("Error opening file %s: %v\n", groupFile, err)
```
Comment (worldwise001) [inline ownership.go]: groupFile should be fileName?

File: ownership.go [hunk 2]
```diff
@@ -50,40 +49,34 @@ func NewOwnership(username, groupname string) (Ownership, error) {
 
 // lookupUID resolves a username to a numeric id.
 func lookupUID(username string) (uint32, error) {
-	u, err := user.Lookup(username)
+	gid, err := lookupIDInFile(username, passwdFile)
```
Comment (worldwise001) [inline ownership.go]: gid should be uid?

File: ownership.go [hunk 3]
```diff
@@ -92,5 +85,5 @@ func lookupGidInFile(groupname string, file *os.File) (uint32, error) {
 		}
 	}
 
-	return 0, errors.New("no such group")
+	return 0, fmt.Errorf("%s not found", name)
```
Comment (worldwise001) [inline ownership.go]: I'd maybe add fileName into this error message.

Comment (mcpherrinm) [inline ownership.go]: The parent error says whether it's a uid or a gid

**Pull request merged**